README.md

```diff
@@ -163,6 +163,67 @@ IList<string> documentsToDelete = new List<string> {
 client.DocumentManager.AddOrUpdateDocuments(sourceId, documentsToAdd, documentsToDelete, null);
 ```
 
+## Using the SDK to push documents to the Stream API
+
+Methods to interact with the Stream API are part of the `client.DocumentManager` object. It’s a different mode of operations from the Push API, but very similar, with several more calls.
+
+### Requirements:
+
+* Before you proceed with this method, ensure you’ve understood how to [index data](https://docs.coveo.com/en/2956/coveo-for-commerce/index-commerce-catalog-content-with-the-stream-api) and [create a commerce catalog](https://docs.coveo.com/en/3139/coveo-for-commerce/create-a-coveo-commerce-catalog).
+* For additional information and payload examples, refer to [How to Stream Your Catalog Data to Your Source](https://docs.coveo.com/en/lb4a0344/coveo-for-commerce/how-to-stream-your-catalog-data-to-your-source).
+* To use the Stream API, a version of the CoveoPlatformConfig needs to be created where useStreamApi is set to true:
+`new CoveoPlatformConfig(Constants.Endpoint.UsEast1.PROD_PUSH_API_URL, Constants.PlatformEndpoint.UsEast1.PROD_PLATFORM_API_URL, apiKey, organizationId, true)`
+
+### How the Stream API works
+
+The Stream API works in two ways:
+
+* Stream mode (a full rebuild)
+* Update mode (an incremental refresh)
+
+### Call Wrappers for the Stream API
+
+Call wrappers for the Stream API are found in the StreamApiDocumentServiceManager. You will need to create the [catalog source](https://docs.coveo.com/en/3295/index-content/add-or-edit-a-catalog-source) in the Admin UI first. Batching documents is recommended.
```

Comment:
Should I link "Batching documents" to here for context?https://docs.coveo.com/en/90/index-content/manage-batches-of-items-in-a-push-source?